webapp/src/main/java/com/box/l10n/mojito/service/leveraging/LeveragerByContentAndRepository.java

```diff
@@ -15,7 +16,7 @@
  *
  * @author garion
  */
-@Component
+@Configurable
```

Comment:
revert?

Reply:
Needed - as we new() up this class in order to be able to inject the repository search params in the contructor - and the main matching method relies on this class from AbstractLeverager being initialized correctly (non-null).
```
@Autowired
    protected TextUnitSearcher textUnitSearcher;
```

Reply:
oh that's from another PR?